### PR TITLE
Projects: Project list doesn’t show correct status

### PR DESCRIPTION
**Pull request by daniel-moderiano**
This PR addresses the following acceptance criteria regarding the titled issue:
- The color of the status badge matches the project status
- The text writes “Critical” for the error status and “Stable” for the info status
- Covered with a Cypress test

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-7)

**Comment by daniel-moderiano** [comment]
Part of this PR involving the UI text function has been refactored in PR #9

File: features/projects/components/project-card/project-card.tsx [hunk 1]
```diff
@@ -21,9 +20,9 @@ const languageNames = {
 };
 
 const statusColors = {
-  [ProjectStatus.stable]: BadgeColor.success,
+  [ProjectStatus.info]: BadgeColor.success,
```
Comment (jkettmann) [inline features/projects/components/project-card/project-card.tsx:23]: Great job!

File: features/projects/components/project-card/project-card.tsx [hunk 2]
```diff
@@ -106,6 +105,21 @@ const ViewIssuesAnchor = styled.a`
 
 export function ProjectCard({ project }: ProjectCardProps) {
   const { name, language, numIssues, numEvents24h, status } = project;
+
+  // Used to switch out the default status text returned from API to a preferred text for the UI
+  const modifyStatusTextForUI = (statusText: string) => {
```
Comment (jkettmann) [inline features/projects/components/project-card/project-card.tsx:110]: Exactly what I thought of. One minor note is that you can move this function outside the component which would make it a constant. It's slightly cleaner IMO although it shouldn't make a difference

Comment (daniel-moderiano) [reply features/projects/components/project-card/project-card.tsx:110]: Great suggestion, I wasn't too sure on where to put functions like this. I like the readability improvements when using it outside though.

**Pull request merged**